src/services/localStorageService.js

```diff
@@ -0,0 +1,34 @@
+export const removeFromHistory = placeId => {
+  const locationsHistory = JSON.parse(localStorage.getItem('locationsHistory')) || [];
```

Comment:
locationsHistory key should be stored as const variable instead of string usage for every LS call